R/encoloc.R

```diff
@@ -31,9 +31,9 @@ xqtl_enrichment_wrapper <- function(gwas_finemapped_data, xqtl_finemapped_data,
     # Process GWAS data
     gwas_pip <- list()
     for (file in gwas_finemapped_data) {
-        gwas_data <- readRDS(file)[[gwas_finemapping_obj]]
+        gwas_data <- if (!is.null(gwas_finemapping_obj)) readRDS(file)[[gwas_finemapping_obj]] else readRDS(file)
         pip <- gwas_data$pip
-        names(pip) <- gwas_data$variant_names
+        #names(pip) <- gwas_data$variant_names #FIXME: need to check the new results from gwas susie_rss
```

Comment:
what you can do is `if (!is.null(gwas_data$variant_names)) names(pip) <- gwas_data$variant_names `?